File: PmProject.API/Models/Project.cs
using System;
using System.Collections.Generic;

namespace PmProject.API.Models
{
    public class Project : BaesClass
    {
        public Guid Id { get; set; }

        public string Name { get; set; }
        public int Status { get; set; }
        public bool IsDelete { get; set; }

        public Guid TemplateServiceOrderId { get; set; }
        public TemplateServiceOrder TemplateServiceOrder { get; set; }
    }
}

File: PmProject-SPA/src/app/_models/project.ts
import { ServiceOrder } from './service-order';
import { TemplateServiceOrder } from './template-service-order';

export class Project {
    id: string;
    name: string;
    status: number;
    templateServiceOrderId: string;
    templateServiceOrde: TemplateServiceOrder;
    serviceOrder: Array<ServiceOrder>;
}